package.json

```diff
@@ -20,19 +19,16 @@
     "test:watch": "npm run karma",
     "nightwatch": "nightwatch",
     "precommit": "lint-staged && npm test",
-    "stylelint":
-      "stylelint 'app/**/*.scss' 'app/**/*.css' 'assets/**/*.scss' 'assets/**/*.css'",
+    "stylelint": "stylelint 'app/**/*.scss' 'app/**/*.css' 'assets/**/*.scss' 'assets/**/*.css'",
     "lint": "eslint app tests && npm run stylelint",
-    "report-coverage":
-      "cat ./coverage/**/*.info | ./node_modules/.bin/coveralls",
+    "report-coverage": "cat ./coverage/**/*.info | ./node_modules/.bin/coveralls",
     "docker:reset": "npm run docker:down && npm run docker:up",
     "docker:up": "docker-compose -f ./docker/dc.selenium.yml up -d",
     "docker:down": "docker-compose -f ./docker/dc.selenium.yml down",
-    "locales:extract":
-      "i18next-extract-gettext --files='./+(app)/**/*.+(js|json)' --output=app/common/locales/nhs.admin.web.pot"
+    "locales:extract": "i18next-extract-gettext --files='./+(app)/**/*.+(js|json)' --output=app/common/locales/nhs.admin.web.pot"
   },
   "lint-staged": {
-    "*.{js,json,css,scss}": ["prettier --write", "git add"]
+    "app/**/*.{js,json,css,scss}": ["prettier --write", "git add"]
```

Comment:
I'm strongly confident that formatter should not be applied only for selected folders, but to entire app codebase. All code which able to be committed — should be passed through formatter.

Reply:
Hm... All codebase is in the `app` directory. 
I don't think that we need to run `prettier` on files like `package.json`, `package-lock.json`, `webpack.config.js` etc. 
If you want to run `prettier` for all files, we need to add `package.json` to ignore or remove `.json` from extensions.

Reply:
The whole idea of using tool like formatter is that developer should not think about code formatting and I think that this principle must be applied to the whole codebase, including configs and other human-editable files